changelog/changes/enough-launch-walked.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "`from_azure_blob_storage` operator"
+type: feature
+authors: raxyte
+pr: 5429
+---
+
+We added a `from_azure_blob_storage` operator that works similarly to `from_file` but supports additional Azure Blob Storage specific options.
```

Comment:
Let's try to not always use "we fixed", "we added", etc. Make it more interesting:

````suggestion
The new `from_azure_blob_storage` operator works similarly to `from_file` but
supports additional Azure Blob Storage specific options. For example, you can
set the `account_key`:

```tql
from_azure_blob_storage "abfs://container/data/*.csv", account_key="your-account-key"
```
````